Internship2022TeamPatri/Internship2022TeamPatri/Controllers/Auth/Login/LogInViewController.swift

```diff
@@ -0,0 +1,163 @@
+    @IBOutlet weak private var passwordTextField: UITextField!
+    @IBOutlet weak private var loginButton: UIButton!
+    @IBOutlet weak private var notRegisteredLabel: UILabel!
+    @IBOutlet weak private var signupButton: UIButton!
+    @IBOutlet weak private var forgotPasswordButton: UIButton!
+    
+    let yourAttributesSignUp: [NSAttributedString.Key: Any] = [
+          .underlineStyle: NSUnderlineStyle.single.rawValue,
+          .font: UIFont.boldSystemFont(ofSize: 18),
+      ]
+    let yourAttributes: [NSAttributedString.Key: Any] = [
+          .underlineStyle: NSUnderlineStyle.single.rawValue
+      ]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        //Make the keyboard disappear touching anywhere
+        let tapGesture = UITapGestureRecognizer(target: view, action: #selector(UIView.endEditing))
+        view.addGestureRecognizer(tapGesture)
+        
+        // Styling Sign up and forgot password buttons
+        let attributeString = NSMutableAttributedString(
+              string: "Sign up",
+              attributes: yourAttributesSignUp
+           )
+        let attributeStringForgotPassword = NSMutableAttributedString(
+              string: "Forgot password?",
+              attributes: yourAttributes
+           )
+        signupButton.setAttributedTitle(attributeString, for: .normal)
+        forgotPasswordButton.setAttributedTitle(attributeStringForgotPassword, for: .normal)
+        
+        setUpElements()
+    }
+    
+    private func setUpElements() {
+        logInLabel.textColor = .colorText
+        signupButton.tintColor = .colorText
+        forgotPasswordButton.tintColor = .colorText
+        notRegisteredLabel.textColor = .colorText
+        
+        logInLabel.font = UIFont.boldSystemFont(ofSize: 35)
+        signupButton.titleLabel?.font = UIFont.systemFont(ofSize: 22, weight: .bold)
+        
+        Utilities.styleTextField(textfield: emailTextField)
+        Utilities.styleTextField(textfield: passwordTextField)
+        Utilities.styleLogInButton(button: loginButton)
+        Utilities.styleShadowButton(button: loginButton)
+        
+        emailTextField.attributedPlaceholder = NSAttributedString(string: "", attributes: [
+            .foregroundColor: UIColor.black,
+            .font: UIFont.boldSystemFont(ofSize: 16.0)
+        ])
+        passwordTextField.attributedPlaceholder = NSAttributedString(string: "", attributes: [
+            .foregroundColor: UIColor.black,
+            .font: UIFont.boldSystemFont(ofSize: 16.0)
+        ])
+    }
+    
+    @IBAction private func loginTapped(_ sender: Any) {
+        let userEmail = emailTextField.text
+        let userPassword = passwordTextField.text
+        
+        if userEmail == "" || userPassword == "" {
+            alertMessage(userMessage: "All fields are required")
+        } else {
+            let mail = isValidEmail(emailID: userEmail)
```

Comment:
No need for "ID". Be consistent when naming something. If you chose isValidPassword(password:) then isValidEmail(email:) should be a better fit